src/iam.ts

```diff
@@ -1588,8 +1588,6 @@ export class IAM extends IAMBase {
       throw new Error(ERROR_MESSAGES.USER_NOT_LOGGED_IN);
     }
 
-    this._did = parseDID(this._did);
```

Comment:
There is no need to keep such changes in commit history. If possible please try to reset previous commits

Reply:
Yeah, I understand what you mean and I agree but unfortunately, there is no clean way to do that. I'll be more careful next time.